tests/5.0/teams/test_teams_distribute_default_none.F90

```diff
@@ -0,0 +1,97 @@
+!===--- test_teams_distribute_default_none.F90 -----------------------------===//
+!
+! OpenMP API Version 5.0 Nov 2018
+!
+! This tests uses the default(none) clause on a target teams distribute
+! test.
+! The test aims to validate that all values will not have default data
+! sharing
+! attributes.
+!
+!===------------------------------------------------------------------------===//
+
+#include "ompvv.F90"
+
+#define N 128
+
+PROGRAM test_teams_distribute_default_none
+   USE iso_fortran_env
+   USE ompvv_lib
+   USE omp_lib
+   implicit none
+   CHARACTER(len=300):: infoMessage
+
+   OMPVV_TEST_OFFLOADING
+
+   WRITE(infoMessage, *) "Test only uses the default(none) clause and&
+       & does not guarantee that the default(none) is enforced."
+   OMPVV_WARNING(infoMessage)
+
+   OMPVV_TEST_VERBOSE(default_none1() .ne. 0)  !check read/write to private
+   OMPVV_TEST_VERBOSE(default_none2() .ne. 0)  !check atomic update of shared
+   OMPVV_REPORT_AND_RETURN()
+
+CONTAINS
+  INTEGER FUNCTION default_none1()
+    INTEGER :: a(N), b(N), c(N), d(N)
+    INTEGER :: privatized, x, y, errors, num_teams, share
+    errors = 0
+    share = 0
+
+    DO x = 1, N
+       a(x) = 1
+       b(x) = x
```

Comment:
Here a local variable in 'default_none1' is set to `b = [(x, x=0,N)]` — which is fine but ... (continued below)